database/entity/ContributionLinks.ts

```diff
@@ -0,0 +1 @@
+export { ContributionLinks } from './0037-add_contribution_links_table/ContributionLinks'
```

Comment:
```suggestion
export { ContributionLinks } from './0037-add_contribution_links_table'
```

Reply:
https://user-images.githubusercontent.com/1324583/173019011-d2bbc1b5-f08d-4889-aadc-be9a55dd3ab6.mp4

Reply:
Ja das war vorher ein Fehler in der index.ts, da war eine Kopie des String mit dem Postfix /ContributionLinks
In der index.ts darf nur der String stehen './0037-add_contribution_links_table', was ich gestern auch koorigiert und eingecheckt habe.
Könnte es sein, dass du neben dem 1920er Branch auch noch den 1921er auf deinen gemerged hast? Denn dann könnte sich das evtl. wieder überschrieben haben...